lib/cldr/backend.ex

```diff
@@ -171,12 +171,15 @@ defmodule Cldr.Territory.Backend do
             "gbcma", "gbcmd", "gbcmn", "gbcon", "gbcov", "gbcrf", "gbcry", "gbcwy",
             "gbdal", "gbdby", "gbden", ...]}
 
+            => #{inspect __MODULE__}.known_territory_subdivisions(:AI)
+            {:ok, nil}
+
             iex> #{inspect __MODULE__}.known_territory_subdivisions(:ZZZ)
             {:error, {Cldr.UnknownTerritoryError, "The territory :ZZZ is unknown"}}
 
         """
         @doc since: "2.2.0"
-        @spec known_territory_subdivisions(atom() | String.t() | LanguageTag.t()) :: {:ok, String.t()} | {:error, {module(), String.t()}}
+        @spec known_territory_subdivisions(atom() | String.t() | LanguageTag.t()) :: {:ok, String.t()} | {:ok, nil} | {:error, {module(), String.t()}}
```

Comment:
It looks like I missed `{:ok, [String.t()]}` in the spec as well, and we could write the spec as: `{:ok, [String.t()] | nil}`. 

Do you have a use case where it would be more helpful to have it return a list instead of `nil`? 

I'm on the fence because the `nil` value indicates no territory_code in `Cldr.known_territory_subdivisions`, but I also see the value in the list if the result is used for matching something like `code in subdivisions` which would raise on `nil`.

cc @kipcole9

Reply:
Nice catch! Updated.

Yep. We want to list the known subdivisions for a country as an input in a form. Tbh it's not too big a hassle, it is more convenient to just say `code in subdivision` but we can also just extract it into a method and pattern match the nil to an empty list.

Reply:
Awesome, there is also [List.wrap/1](https://hexdocs.pm/elixir/List.html#wrap/1).